source/particle/world.cc

```diff
@@ -1299,6 +1277,15 @@ namespace aspect
       }
       prm.leave_subsection ();
 
+      if (print_timing_output)
+        particle_timer.reset(&this->get_computing_timer());
+      else
+        particle_timer.reset(new TimerOutput(const_cast<ConditionalOStream &>(this->get_pcout()),
+                                             TimerOutput::never,
+                                             TimerOutput::wall_times));
```

Comment:
I meant to say in all of these places above that you should just generically replace all occurrences of `particle_timer` by `this->get_computing_timer()`. But I suppose you do things the current way because you just want to (i) not incur the overhead of the `MPI_Barrier` if you're not interested in output, and (ii) just want to pipe output into nirvana in that case by never generating output in the first place. Is this correct? If so, maybe document this in the documentation of the `particle_timer` variable.